tests/h/util/markdown_test.py

```diff
@@ -12,78 +12,74 @@ def test_it_ignores_math_block(self):
         actual = markdown.render("$$1 + 1 = 2$$")
         assert actual == "<p>$$1 + 1 = 2$$</p>\n"
 
-    def test_it_ignores_inline_match(self):
+    def test_it_ignores_inline_math(self):
         actual = markdown.render(r"Foobar \(1 + 1 = 2\)")
         assert actual == "<p>Foobar \\(1 + 1 = 2\\)</p>\n"
 
-    def test_it_sanitizes_the_output(self, markdown_render, sanitize):
-        markdown.render("foobar")
-        sanitize.assert_called_once_with(markdown_render.return_value.return_value)
-
-    @pytest.fixture
-    def markdown_render(self, patch):
-        return patch("h.util.markdown._get_markdown")
-
-    @pytest.fixture
-    def sanitize(self, patch):
-        return patch("h.util.markdown.sanitize")
-
-
-class TestSanitize:
     @pytest.mark.parametrize(
         "text",
         [
-            '<a href="mailto:foo@example.net">example</a>',  # Don't add rel and target attrs to mailto: links
-            '<a title="foobar">example</a>',
-            '<a href="https://example.org" rel="nofollow noopener" target="_blank" title="foobar">example</a>',
+            '<p><a href="mailto:foo@example.net">example</a></p>',  # Don't add rel and target attrs to mailto: links
```

Comment:
Since the test is now calling `render()` instead of calling `_sanitize()` directly Markdown would add `<p>...</p>`'s to the text if they weren't there already. I just pre-added the `<p>`'s to the input-to-be-santizied in the test as the easiest way to keep the test passing.